openwisp_radius/settings.py

```diff
@@ -68,7 +68,8 @@ def get_settings_value(option, default):
 PASSWORD_RESET_URLS = {
     # fallback in case the specific org page is not defined
     'default': (
-        'https://example.com/{organization}/password/confirm/reset/{uid}/{token}'
+        'http://{domain}/api/v1/{organization}/account/password/reset/confirm/?'
+        'uid={uid}&token={token}'
```

Comment:
this should be `https://example.com/{organization}/password/reset/confirm/{uid}/{token}`

Reply:
@nemesisdesign  I think this should be `https://example.com/{organization}/password/reset/confirm/?uid={uid}&token={token}`

This is so that the frontend can extra the `uid` and `token` from the link and send it with the new password to the backend. if we don't do that, how will the frontend know the `uid` and `token` to send to the user?

Reply:
@NoumbissiValere it should default to the URL used by [openwisp-wifi-login-pages](https://github.com/openwisp/openwisp-wifi-login-pages/) to perform the last step of this process, which allows the user to change their password (which is the URL I suggested, but please double check).
If a user uses a different system they'll be smart enough to know they have to change this setting.

Reply:
Also, please make it default to HTTPS, but set this setting in the development environment (`test/settings.py`) to use `http://localhost:8080`.

Reply:
@nemesisdesign please if we are doing this, then what will be the use of `domain` from site

Reply:
to use as prod default